heartbeat/aliyun-vpc-move-ip

```diff
@@ -37,27 +54,37 @@ USAGE="usage: $0 {start|stop|status|meta-data}";
 #
 ###############################################################################
 
+request_create_route_entry() {
+	cmd="${OCF_RESKEY_aliyuncli} vpc CreateRouteEntry --RouteTableId $OCF_RESKEY_routing_table --DestinationCidrBlock ${OCF_RESKEY_address}/32 --NextHopId $ECS_INSTANCE_ID --NextHopType Instance ${ENDPOINT}"
+	ocf_log debug "executing command: $cmd"
+	$cmd
+}
+
+request_delete_rote_entry() {
```

Comment:
You got a typo here in "route" (and where it's called below)

Reply:
you are right，tks